app/src/main/java/org/fossasia/openevent/general/welcome/WelcomeFragment.kt

```diff
@@ -26,12 +24,15 @@ import org.koin.androidx.viewmodel.ext.android.viewModel
 
 const val LOCATION_PERMISSION_REQUEST = 1000
 const val WELCOME_FRAGMENT = "welcomeFragment"
+const val PREFERENCE_KEY = "experimental_warning_shown"
```

Comment:
```suggestion
const val EXPERIMENTAL_WARNING_PREFERENCE_KEY = "experimental_warning_shown"
```

Reply:
Please always use explicit variable names, not generic names like `PREFERENCE_KEY`